local-macros/src/lib.rs

```diff
@@ -1026,3 +1026,366 @@ pub fn type_type(attr: TokenStream, item: TokenStream) -> TokenStream {
     }
     .into()
 }
+
+#[derive(Debug, FromMeta)]
+struct SymbolTypeArgs {
+    #[darling(default)]
```

Comment:
What does the `default` do here? Default to `None`?

Reply:
I think so